src/Pages/Basket/Basket.scss

```
@@ -0,0 +1,278 @@
+                  padding-top: 12px;
+                  // border: 1px solid rgb(25, 155, 51);
+                }
+                .bk_check {
+                  width: 30px;
+                }
+                .bk_img {
+                  width: 100px;
+                }
+                .bk_pdInfo {
+                  width: 500px;
+                }
+                .bk_price {
+                  width: 100px;
+                }
+                .bk_count {
+                  width: 80px;
+                }
+
+                .bk_plus {
+                  width: 100px;
+                }
+                .bk_dilivery_type {
+                  width: 100px;
+                }
+                .bk_dilivery {
+                  width: 90px;
+                }
+                .bk_sum {
+                  width: 100px;
+                }
+                .bk_choose {
+                  width: 100px;
+                }
+              }
+
+              .basket_items {
+                // Item components
+                .item {
+                  display: flex;
+                  margin: 20px 0px;
+                  height: 100px;
+                  border-bottom: 1px solid rgb(226, 224, 224);
+                  .item_img {
+                    width: 70px;
+                    height: 70px;
+                  }
+                  .bk_check {
+                    width: 30px;
+                  }
+                  .bk_img {
+                    width: 100px;
+                  }
+                  .bk_pdInfo {
+                    width: 500px;
+                  }
+                  .bk_price {
+                    width: 100px;
+                  }
+                  .bk_count {
+                    display: flex;
+                    width: 80px;
+                    input {
+                      width: 23px;
+                      height: 20px;
+                      margin-right: 2px;
+                    }
+                    .count_box {
+                      display: flex;
+                      flex-direction: column;
+
+                      .fa-sort-up,
+                      .fa-sort-down {
+                        background-color: white;
+                        border: 0.8px solid gray;
+                        width: 20px;
+                        height: 10px;
+                      }
+                    }
+                  }
+                  .bk_plus {
+                    width: 100px;
+                  }
+                  .bk_dilivery_type {
+                    width: 100px;
+                  }
+                  .bk_dilivery {
+                    width: 90px;
+                  }
+                  .bk_sum {
+                    width: 100px;
+                  }
+                  .bk_choose {
+                    display: flex;
+                    flex-direction: column;
+                    align-items: center;
+                    width: 100px;
+                    button {
```

```suggestion
                    width: 100px;
                    
                    button {
```
새로운 셀렉터 앞에서는 꼭 한줄 띄워주세요. 가독성을 위해서!